package.json

```diff
@@ -45,7 +45,7 @@
     "font-awesome": "^4.7.0",
     "lodash.uniqby": "^4.7.0",
     "mailto-link": "^1.0.0",
-    "prop-types": "^15.8.1",
+    "prop-types": "^15.7.2",
```

Comment:
Why do we need to downgrade `prop-types` version?

Reply:
We don't need to downgrade prop-types, fixed